packages/data-stores/src/help-center/actions.ts

```diff
@@ -153,7 +159,9 @@ export const setShowHelpCenter = function* (
 		yield setShowMessagingWidget( false );
 	}
 
+	yield setMessage( options.searchTerm );
```

Comment:
Setting the term without checking if it is not empty is necessary if a page has two different entry points.